Cargo.toml

```diff
@@ -8,18 +8,17 @@ repository = "https://github.com/onomyprotocol/web30.git"
 version = "0.18.3"
 
 [dependencies]
-clarity = "0.5"
-futures = "0.3"
-hyper = {version = "0.14", features = ["full"]}
+clarity = { git = "https://github.com/AaronKutch/clarity.git", rev = "c50d7dca265d0b29da2326cae1b2996dc27b5ee1" }
+hyper = { version = "0.14", features = ["full"] }
 hyper-tls = "0.5.0"
 lazy_static = "1.4"
 log = "0.4"
 num = "0.4"
-num256 = "0.3"
 serde = "1.0"
 serde_derive = "1.0"
 serde_json = "1.0"
-tokio = {version = "1.4", features = ["macros", "time"]}
+tokio = { version = "1.4", features = ["macros", "time"] }
```

Comment:
Don't we need to update the "tokio" version ?

Reply:
good catch